emission/core/wrapper/suggestion_sys.py

```diff
@@ -13,6 +13,7 @@
 import os
 import emission.net.ext_service.geocoder.nominatim as geo
 from emission.net.ext_service.geocoder.nominatim import Geocoder
+import emission.storage.decorations.trip_queries as tripqu
```

Comment:
nit: wait, why do you even need to import this? It is not used anywhere; you use the timeseries directly,